.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -43,7 +43,7 @@ body:
     label: Component Name
     description: >-
       Write the short name of the module, plugin, task or feature below,
```

Feature? That opens a very large range of interpretations.

---

I think this is a leftover from ansible-core (when it still had all the plugins and modules in it). What do you think about:
```suggestion
      Write the short name of the module or plugin, or which other part(s) of the collection this feature affects.
```

---

That's better, yes.